test/test_formatters.rb

```diff
@@ -63,6 +63,30 @@ def test_vim
     ))
   end
 
+  def test_vim_with_line_number
+    vim = formatter_for(:format => 'vim', :line_number => true)
+    assert_equal %{C\t./script.rb\t/^class C < D$/;"\tc\tline:1\tclass:A.B\tinherits:D}, vim.format(build_tag(
```

Comment:
We don't need this many tests with variations of different tag types like class vs. module vs. method, etc. These variations are all tested in other tests. Here, we want to focus on line numbers. So instead, lets test the output for a tag that has `:line => 1`, then another line number `:line => 18`, for example, then a large line number such as `:line => 123456`. Nothing else other than line number needs to change in these tag definitions, to keep the tests as focused as possible.